src/wallet/rpc/encrypt.cpp

```diff
@@ -220,7 +220,11 @@ RPCHelpMan encryptwallet()
                 "After this, any calls that interact with private keys such as sending or signing \n"
                 "will require the passphrase to be set prior the making these calls.\n"
                 "Use the walletpassphrase call for this, and then walletlock call.\n"
-                "If the wallet is already encrypted, use the walletpassphrasechange call.\n",
+                "If the wallet is already encrypted, use the walletpassphrasechange call.\n"
+                "** IMPORTANT **\n"
+                "For security reasons, the encryption process will generate a new HD seed, leading to the creation of a fresh set of active descriptors. "
+                "Therefore, it is crucial to replace any existing backups of your wallet file with the newly generated, encrypted wallet file.\n"
```

Comment:
```suggestion
                "Therefore, it is crucial to replace any existing backups of your wallet file with the newly generated wallet file, containing the new encrypted keys.\n"
```

I read it as the wallet file itself being encrypted.
I'd tweak the wording a bit to prevent confusion.

Reply:
Sure. Done as suggested.